x-pack/legacy/plugins/siem/public/components/charts/common.tsx

```diff
@@ -138,10 +139,10 @@ export const getTheme = () => {
       barsPadding: 0.5,
     },
   };
-  const isDarkMode = chrome.getUiSettingsClient().get('theme:darkMode');
+  const isDarkMode = chrome.getUiSettingsClient().get(DEFAULT_DARK_MODE);
```

Comment:
Looks like `isDarkMode` is showing up as an `any`, should we type it as a boolean?

Reply:
Yea, I will type it while I'm in there.